2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex

```diff
@@ -218,7 +218,7 @@ \subsection{Sobol Indices}
          \label{varCond}
          \Expect{Y|\vect{U}} = \Expect{Y|\vect{X}}
        \end{equation}
-       As a matter of fact : $\Expect{Y|\vect{U}} = \Expect{Y|\psi(\vect{X})}$ is the orthogonal projection in a $L_2$ sense of $Y$ on the space generated by $\psi(\vect{X})$, which is the same as the one generated by $\vect{X}$, thus the equality of the random variables  (\ref{varCond}).\\
+       As a matter of fact : $\Expect{Y|\vect{U}} = \Expect{Y|\psi(\vect{X})}$ is the orthogonal projection in a $L_2$ sens of $Y$ on the space generated by $\psi(\vect{X})$, which is the same as the one generated by $\vect{X}$, thus the equality of the random variables  (\ref{varCond}).\\
```

Comment:
here too

Reply:
I cannot understand this: it is as if a Gremlins were to insert new bugs! Fixed.